corehq/apps/dump_reload/tests/test_couch_dump_load.py

```diff
@@ -218,10 +218,11 @@ def test_dump_toggles(self):
         output_stream = BytesIO()
 
         with mock_out_couch(docs=[doc.to_json() for doc in self.mocked_toggles.values()]):
-            dump_counter = dumper.dump(output_stream)
-        self.assertEqual(3, dump_counter['Toggle'])
+            dumper.dump(output_stream)
         output_stream.seek(0)
-        dumped = [json.loads(line.strip()) for line in output_stream.readlines()]
+        lines = output_stream.readlines()
+        dumped = [json.loads(line.strip()) for line in lines]
+        self.assertEqual(len(dumped), 3, ','.join([d['slug'] for d in dumped]))
```

Comment:
what does the third argument here do?

Reply:
it's used as the assertion message in case of failuer